### PR TITLE
Mary had a little bug, little bug...

### PR DESCRIPTION
**Pull request by fgeorgatos**

### CODE REVIEW

File: docs/user/firststeps.rst [hunk 1]
```diff
@@ -232,7 +232,7 @@ For example, if you want to keep the new notebook(s), run the following.
 .. code-block:: console
 
     git add notebooks # track everything inside the notebooks folder
-    git commit -m "Added some notebooks"
+    git commit -m "Added some notebooks" -a
```
Comment (rokroskar) [inline docs/user/firststeps.rst]: why?

**Comment by rokroskar** [comment]
I don't understand what problem this PR is solving, can you clarify please?

File: docs/user/firststeps.rst (hunk 1 above)
Comment (fgeorgatos) [reply docs/user/firststeps.rst]: Well, I was forced to use it to make it advance, but now that I see it again, wait before merging:
* let's try reproduce it elsewhere, to rule out an artifact on my end - or is it prereq for the dir?
(interestingly, I didn't need to use it a bit later on in the 1st steps instructions, in another git op)

Comment (rokroskar) [reply docs/user/firststeps.rst]: what do you mean "to make it advance"? Are you sure you didn't forget the line above (`git add notebooks`)?

Comment (fgeorgatos) [reply docs/user/firststeps.rst]: @rokroskar : I'm sure I did not forget that line - in fact, that was a prerequisite for `-a` to work!

**Comment by rokroskar** [comment]
Can you please give an example of the error you were getting?

**Comment by leafty** [comment]
Closing as a stale PR

**Pull request closed without merging**